app/admin/flags.rb

```diff
@@ -32,7 +32,10 @@
         flag.project_submission.user.username
       end
       row :lesson do
-        link_to flag.project_submission.lesson.title, flag.project_submission.lesson, target: '_blank'
+        link_to flag.project_submission.lesson.title, path_course_lesson_path(flag.project_submission.lesson.path,
```

Comment:
Just a nitpick, can we make the arguments multiline for this instead please?

```
link_to( 
  flag.project_submission.lesson.title,
  path_course_lesson_path(
    flag.project_submission.lesson.path,
    flag.project_submission.lesson.course,
    flag.project_submission.lesson
  ),
  target: '_blank'
)
```

Sorry, Rubocop probably insisted on indenting it. It's so much more readable and atheistical pleasing™️  when its multiline though 😆